deeplearning4j/deeplearning4j-dataimport-solrj/pom.xml

```diff
@@ -136,18 +147,36 @@
                     <version>${nd4j.version}</version>
                     <scope>test</scope>
                 </dependency>
+                <dependency>
+                    <groupId>org.deeplearning4j</groupId>
+                    <artifactId>deeplearning4j-cuda-11.0</artifactId>
```

Comment:
You've got to make this dynamic too.

Reply:
This is mainly for the version, not the artifactIds. That's handled by change-cuda-versions.sh.